File: Class-and-Structure/class_and_structure.py
class Person:
    def __init__(self):
        self.name = ""
        self.age = 0
    def print_profile(self):
        print(f"Name: {self.name}")
        print(f"Age: {self.age}")
    def add_age(self):
        self.age += 1

class ProductInfo:
    def __init__(self):
        self.name = ""
        self.price = 0
        self.comments = [""]

def main():
    person = Person()
    person.name = "Mr. Qiita"
    person.age = 15
    person.print_profile()
    product = ProductInfo()
    product.name = "Product A"
    product.price = 1000
    product.comments = ["Good", "About IT"]
    print(product.price * 5)

if __name__=="__main__":
    main()

File: Class-and-Structure/class_and_structure.cpp
#include <iostream>
#include <string>
using namespace std;

class Person
{
public:
    string name;
    int age;
    void print_profile()
    {
        cout << "Name: " << name << "\nAge: " << age << endl;
    }
    void add_age()
    {
        age++;
    }
};

typedef struct productinfo
{
    string name;
    int price;
    string comments[50];
} productinfo;

int main(void)
{
    Person person;
    person.name = "Mr. Qiita";
    person.age = 15;
    person.print_profile();
    productinfo product;
    product.name = "Product A";
    product.price = 1000;
    product.comments[0] = "Good";
    product.comments[1] = "About IT";
    cout << product.price * 5 << endl;
    return 0;
}